tests/integration/tier0/non-destructive/modified-releasever/test_modified_releasever.py

```diff
@@ -1,83 +1,78 @@
-def system_release_backup(shell, backup_directory):
-    """
-    Fixture.
-    Backup /etc/system-release before the test makes modifications to it.
-    Restore the file after the test
-    """
-    # Backup /etc/system-release
-    backup_file = os.path.join(backup_directory, "system-release.bkp")
-    assert shell(f"cp /etc/system-release {backup_file}").returncode == 0
-
-    yield
-
-    # Restore system-release
-    assert shell(f"mv -f {backup_file} /etc/system-release").returncode == 0
-
-
-def test_inhibitor_releasever_noexistent_release(convert2rhel, config_at, os_release, shell, system_release_backup):
+    config_file_name = f"{SystemInformationRelease.distribution}-{SystemInformationRelease.version.major}-x86_64.cfg"
+    shell(f"sed -i 's/releasever=.*/releasever=420/g' /usr/share/convert2rhel/configs/{config_file_name}")
+    with convert2rhel(
+        "analyze -y --serverurl {} --username {} --password {} --debug".format(
+            TEST_VARS["RHSM_SERVER_URL"],
+            TEST_VARS["RHSM_SCA_USERNAME"],
+            TEST_VARS["RHSM_SCA_PASSWORD"],
+        ),
+        unregister=True,
+    ) as c2r:
+        c2r.expect("--releasever=420")
+        c2r.expect_exact("ERROR - (ERROR) ENSURE_KERNEL_MODULES_COMPATIBILITY::PROBLEM_WITH_PACKAGE_REPO")
+
+    assert c2r.exitstatus == 0
+
+
+@pytest.mark.parametrize("backup_files", ["/etc/system-release"], indirect=True)
+def test_inhibitor_releasever_noexistent_release(backup_files, convert2rhel, shell):
     """
     Verify that running not allowed OS release inhibits the conversion.
-    Modify the /etc/system-release file to set the releasever to an unsupported version (e.g. x.1.1111)
+    Modify the /etc/system-release file to set the releasever to an unsupported version (e.g. x.11.1111)
     """
-    with config_at(Path("/etc/system-release")).replace_line(
-        "release .+",
-        f"release {os_release.version[0]}.11.1111",
-    ):
-        with convert2rhel(
-            "-y --serverurl {} --username {} --password {} --debug".format(
-                TEST_VARS["RHSM_SERVER_URL"],
-                TEST_VARS["RHSM_SCA_USERNAME"],
-                TEST_VARS["RHSM_SCA_PASSWORD"],
-            ),
-            unregister=True,
-        ) as c2r:
-            c2r.expect(
-                f"CRITICAL - {os_release.name} of version {os_release.version[0]}.11 is not allowed for conversion."
-            )
-        assert c2r.exitstatus == 1
+    shell(
+        f"sed -i 's/release {SystemInformationRelease.version.major}.{SystemInformationRelease.version.minor}/release {SystemInformationRelease.version.major}.11.11111/' /etc/system-release"
+    )
+    with convert2rhel(
+        "analyze -y --serverurl {} --username {} --password {} --pool {} --debug".format(
+            TEST_VARS["RHSM_SERVER_URL"],
+            TEST_VARS["RHSM_USERNAME"],
+            TEST_VARS["RHSM_PASSWORD"],
+            TEST_VARS["RHSM_POOL"],
+        ),
+        unregister=True,
+    ) as c2r:
```

Comment:
Why do you change the SCA account to the non-SCA?

Reply:
right, this is due to some bad rebase I have done. The PR was created before this SCA change and when I rebased I had to do some mistake. Thanks for noticing that!